### PR TITLE
Stua0000/issue487 change viewpoint

### PR DESCRIPTION
**Pull request by macca400**
@anttikajanus Can you review please?

### CODE REVIEW

File: src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/ChangeViewpoint/ChangeViewpoint.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+using Esri.ArcGISRuntime.Geometry;
+using System;
+using System.Collections.Generic;
+using System.Windows;
+
+namespace ArcGISRuntime.Desktop.Samples.ChangeViewpoint
+{
+    public partial class ChangeViewpoint
+    {    
+        static MapPoint LondonCoords = new MapPoint(-13881.7678417696, 6710726.57374296, SpatialReference.Create(102100));
```
Comment (anttikajanus) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/ChangeViewpoint/ChangeViewpoint.xaml.cs]: Use SpatialReferences enumeration if possible.

Comment (anttikajanus) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/ChangeViewpoint/ChangeViewpoint.xaml.cs]: Use private variables instead.

File: src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/ChangeViewpoint/ChangeViewpoint.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        {
+            try
+            {
+                var viewpoint = new Esri.ArcGISRuntime.Viewpoint(EdinburghEnvelope);
+                //Animates the changing of the viewpoint givng a smooth trasistion from the old to the new view
+                MyMapView.SetViewpointAsync(viewpoint, System.TimeSpan.FromSeconds(5));
+            }
+            catch(Exception ex)
+            {
+                var errorMessage = $"Viepoint could not be set. \n{ex.Message}";
```
Comment (anttikajanus) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/ChangeViewpoint/ChangeViewpoint.xaml.cs]: Remove C# 6.0 functionality

File: src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/ChangeViewpoint/ChangeViewpoint.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            
+        }
+
+        private async void OnRotateButtonClick(object sender, RoutedEventArgs e)
+        {
+            try
+            {
+                var currentRotation = MyMapView.Rotation;
+                //Rotates the viewpoint by the given number of degrees 
+                await MyMapView.SetViewpointRotationAsync(currentRotation + 90.00);
```
Comment (anttikajanus) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/ChangeViewpoint/ChangeViewpoint.xaml.cs]: Comment why 90 is added to the value.

**Comment by anttikajanus** [comment]
Also make sure that you clean up the file from empty lines etc.

File: src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/ChangeViewpoint/ChangeViewpoint.xaml.cs (hunk 2 above)
Comment (katydalton) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/ChangeViewpoint/ChangeViewpoint.xaml.cs]: Also run spell check on comments/text.

**Pull request merged**